### DISCUSSION TITLE
Prime benchmark on SMB - why not also on microcontrollers?

### DISCUSSION THREAD

**Discussion by PeterQuinn925** [PlummersSoftwareLLC/Primes#971] [Ideas]
I enjoyed the video with the benchmark of various Raspberry Pis and other computers. I have two suggestions. One, repeat the exercise on microcontrollers such as arduino/AVR, PI Zero, PI Pico/RP2040, and ESP32. If I were doing it, I'd do all the microcontrollers using the Arduino toolchain.

Second when reporting results, compute the results per $ for a more fair fight.

Thanks for all your videos.

**Comment by rbergen** [comment]
I'm converting this to an Ideas discussion.

**Reply by rbergen** [reply]
...and moved it to the Primes repository as it concerns the drag race much more than it does the results viewer that is PrimeView.
